### PR TITLE
Refact echoscp

### PR DESCRIPTION
**Pull request by dwikler**
This merge request addresses part of issue #85.
It refactors echoscp.py from the tdwii_plus_examples/TDWII_PPVS_subscriber folder.
The EchoSCP class is refactored as CEchoSCP and moved to tdwii_plus_examples package folder in cechoscp.py 
The main function is refactored as an echoscp.py CLI application and moved to the tdwii_plus_examples/cli folder.

### CODE REVIEW

File: tdwii_plus_examples/tests/test_cechoscp.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import unittest
+import logging
+from logging.handlers import MemoryHandler
+import subprocess
+import time
+import re
+from tdwii_plus_examples.cechoscp import CEchoSCP
+
+
+class TestBaseSCP(unittest.TestCase):
```
Comment (sjswerdloff) [inline tdwii_plus_examples/tests/test_cechoscp.py:10]: I think this should be class TestCEchoSCP

File: tdwii_plus_examples/cli/echoscp.py
Comment (sjswerdloff) [inline tdwii_plus_examples/cli/echoscp.py:1]: do we want to add a test of the cli version in the future?
The underlying class is getting tested, so I don't think it's critical unless the cli version is intended to get significant use.

**Review by sjswerdloff — CHANGES_REQUESTED** [review]
Please change class TestBaseSCP in test_echoscp.py to TestCEchoSCP

File: tdwii_plus_examples/tests/test_cechoscp.py (hunk 1 above)
Comment (sjswerdloff) [reply tdwii_plus_examples/tests/test_cechoscp.py]: I took the liberty of changing the name of the class, hope that's OK.

**Review by sjswerdloff — APPROVED** [review]

File: tdwii_plus_examples/cli/echoscp.py
Comment (dwikler) [reply tdwii_plus_examples/cli/echoscp.py:1]: I will have to do it for the next steps cli apps so I will add it at the same time.

**Pull request merged**